src/mpid/ch4/src/ch4_init.h

```diff
@@ -200,6 +200,15 @@ MPL_STATIC_INLINE_PREFIX int MPID_Init(int *argc,
     MPIDI_COMM(MPIR_Process.comm_self, local_map).mode = MPIDI_RANK_MAP_NONE;
     MPIDIU_avt_add_ref(0);
 
+#ifdef MPL_USE_DBG_LOGGING
+    int counter_;
+    if (size < 16) {
+        for (counter_ = 0; counter_ < size; ++counter_) {
+            MPIDIU_comm_rank_to_av(MPIR_Process.comm_world, counter_);
+        }
+    }
+#endif
```

Comment:
Yes, just to do dummy translation to print the addresses into the log. It gives us two things: 1) allowing us to confirm the correctness of rank-address(global rank) mapping; 2) address of addr, which is useful to cross check other part of the debug log.